src/main.cpp

```diff
@@ -0,0 +1,61 @@
+
+#include <controller_manager/controller_manager.h>
+#include <forque_sensor_hardware/ForqueSensorHW.h>
+#include <geometry_msgs/WrenchStamped.h>
+#include <iostream>
+#include <ros/callback_queue.h>
+#include <ros/ros.h>
+
+bool shouldShutdown = false;
+
+void mySigintHandler(int sig) { shouldShutdown = true; }
+
+int main(int argc, char **argv) {
+  // Hardcoded values
+  std::string address = "192.168.1.1";
+  float update_rate_hz = 100;
+
+  // Starting ROS
+  ROS_INFO("Starting ROS node.");
+  ros::init(argc, argv, "forque_hardware_interface",
+            ros::init_options::NoSigintHandler);
+  ros::NodeHandle nh;
+  ros::CallbackQueue queue;
+  nh.setCallbackQueue(&queue);
+  signal(SIGINT, mySigintHandler);
+
+  // Starting other stuff
+  forqueSensorHW::ForqueSensorHW forqueSensorHW("forqueSensor",
+                                                "endEffectorFrameId", address);
```

Comment:
I wouldn't use `endEffectorFrameId` here. Also, does it make sense to use hardcoded `address`? Should we read these from command line args?